src/Helper/ManufacturerHelper.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace ElasticExportCheck24DE\Helper;
+
+use Plenty\Modules\Item\Manufacturer\Contracts\ManufacturerRepositoryContract;
+use Plenty\Modules\Item\Manufacturer\Models\Manufacturer;
+
+/**
+ * Class ManufacturerHelper
+ * @package ElasticExportCheck24DE\Helper
+ */
+class ManufacturerHelper
+{
+	const MAX_CACHE_SIZE = 100;
+
+	/** @var ManufacturerRepositoryContract */
+	private $manufacturerRepository;
+
+	/** @var array */
+	private $manufacturerCache;
+
+	/**
+	 * ManufacturerHelper constructor.
+	 * @param ManufacturerRepositoryContract $manufacturerRepository
+	 */
+	public function __construct(ManufacturerRepositoryContract $manufacturerRepository)
+	{
+		$this->manufacturerRepository = $manufacturerRepository;
+		$this->manufacturerCache = [];
+	}
+
+	/**
+	 * Selects the manufacturer name by ID. Exports the name, if the external name is not configured.
+	 *
+	 * @param int $manufacturerId
+	 * @return string
+	 */
+	public function getName(int $manufacturerId): string
+	{
+		if ($manufacturerId < 0) {
```

Comment:
is 0 a valid ID?

Reply:
No

Reply:
then shouldnt the if say $manufacturerId <= 0 ;P